inc/saiport.h

```diff
@@ -539,6 +539,7 @@ typedef enum _sai_port_attr_t
      * @type sai_uint32_t
      * @flags CREATE_AND_SET
      * @default 0x6A737D
+     * @validonly SAI_PORT_ATTR_SPEED == 25000Mbps, 50000Mbps
```

Comment:
actually validonly flag supports enums and bool's now no uint compare is supported yet 
you can remove this flag or comment it out